libs/components/popovers/src/lib/modules/dropdown/dropdown.component.ts

```diff
@@ -247,6 +242,12 @@ export class SkyDropdownComponent implements OnInit, OnDestroy {
       .subscribe(() => {
         this.#changeDetector.markForCheck();
       });
+
+    this.#contentInfoProvider?.getInfo()?.subscribe((contentInfo) => {
```

Comment:
We originally changed this to use the async pipe in the template intentionally, I think because it was less observable management that way